client/lib/explat/index.ts

```diff
@@ -0,0 +1,37 @@
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks'
```

Comment:
This is the `/package` code being initialised for Calypso.

Mostly stubs at the moment, but straightforward to complete.